doc/Language/functions.pod

```diff
@@ -674,6 +674,20 @@ class LoggedVersion is Version {
 say LoggedVersion.new('1.0.2');
 =end code
 
+If you need to make multiple calls to the wrapped code or to gain a reference to
+it, such as for performing introspection on the it, you can use C<nextcallee>.
```

Comment:
Small typo here to fix: `on the it`. :blush: 

To go further...perhaps `such as for performing introspection on it` would read better as `such as performing introspection`.